Tests/NSInvocationUtilsTest.m

```diff
@@ -92,113 +92,114 @@ - (void)testInvokeWithDelay {
 	[[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
-	GHAssertTrue(invokeTesting3Called_, @"Method was not called");
+	XCTAssertTrue(invokeTesting3Called_, @"Method was not called");
 }
 
 - (void)_invokeTesting3:(NSNumber *)number1 {
-	GHAssertEqualObjects([NSNumber numberWithInteger:1], number1, nil);
+	XCTAssertEqualObjects([NSNumber numberWithInteger:1], number1);
 	invokeTesting3Called_ = YES;
 }
 
 
 - (void)testInvokeProxy {
 	[[self gh_proxyAfterDelay:0.1] _invokeTesting4:1];
 	
-	GHAssertFalse(invokeTesting4Called_, @"Method should be delayed");
+	XCTAssertFalse(invokeTesting4Called_, @"Method should be delayed");
 	[[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
-	GHAssertTrue(invokeTesting4Called_, @"Method was not called");
+	XCTAssertTrue(invokeTesting4Called_, @"Method was not called");
 }
 
 - (void)_invokeTesting4:(NSInteger)n {
-	GHAssertTrue(n == 1, @"Should be equal to 1 but was %d", n);
+	XCTAssertTrue(n == 1, @"Should be equal to 1 but was %ld", (long)n);
 	invokeTesting4Called_ = YES;
 }
 
 - (void)testInvokeOnMainThread {
 	[[self gh_proxyOnMainThread:YES] _invokeTestingMainThread:1];
 	[[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
-	GHAssertTrue(invokeTestingMainThreadCalled_, @"Method was not called");
+	XCTAssertTrue(invokeTestingMainThreadCalled_, @"Method was not called");
 }
 
 - (void)_invokeTestingMainThread:(NSInteger)n {
-	GHAssertTrue([NSThread isMainThread], nil);
-	GHAssertTrue(n == 1, @"Should be equal to 1 but was %d", n);
+	XCTAssertTrue([NSThread isMainThread]);
+	XCTAssertTrue(n == 1, @"Should be equal to 1 but was %ld", (long)n);
 	invokeTestingMainThreadCalled_ = YES;
 }
 
 // TODO(gabe): This isn't a real failure
 - (void)_testInvokeNestedProxy {
 	[[[self gh_proxyOnMainThread:YES] gh_proxyAfterDelay:0.1] _invokeTestingNested:1];
 	
-	GHAssertFalse(invokeTestingNestedCalled_, @"Method should be delayed");
+	XCTAssertFalse(invokeTestingNestedCalled_, @"Method should be delayed");
 	[[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
-	GHAssertTrue(invokeTestingNestedCalled_, @"Method was not called");
+	XCTAssertTrue(invokeTestingNestedCalled_, @"Method was not called");
 }
 
 - (void)_invokeTestingNested:(NSInteger)n {
-  GHTestLog(@"invokeTestingNestedCalled on main thread: %d", [NSThread isMainThread]);
-	GHAssertTrue([NSThread isMainThread], @"Should be on main thread; This is currently an expected failure!");
-	GHAssertTrue(n == 1, @"Should be equal to 1 but was %d", n);
+  NSLog(@"invokeTestingNestedCalled on main thread: %d", [NSThread isMainThread]);
+	XCTAssertTrue([NSThread isMainThread], @"Should be on main thread; This is currently an expected failure!");
+	XCTAssertTrue(n == 1, @"Should be equal to 1 but was %ld", (long)n);
 	invokeTestingNestedCalled_ = YES;
 }
-
+/* This doesn't currently test anything, because XCTests are run on the main thread
```

Comment:
Wasn't 100% comfortable doing so vs. just disabling it; on the other hand, who's going to be touching this code…